src/lint/lint.ts

```diff
@@ -48,37 +51,51 @@ export async function lint(options: Options, config: ProjectConfig) {
     minimumSeverity: Severity.WARNING
   });
 
-
   const analyzer = new Analyzer({
     urlLoader: new FSUrlLoader(config.root),
     urlResolver: new PackageUrlResolver(),
   });
-
   const linter = new lintLib.Linter(rules, analyzer);
 
+  // This is awkward, it would be nice to be able to get access to the analysis
+  // of a lint run rather than doing this sort of parallel construction.
+  // We want the analysis that the linter sees so that we can apply fixes
+  // to the same version of the files that the fixes were constructed for.
```

Comment:
This is very reasonable, can we file an issue to track this & then link it here?

Reply:
Done